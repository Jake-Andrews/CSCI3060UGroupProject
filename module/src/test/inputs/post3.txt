login
USER
post
Toronto
500.00
15
5
logout
exit