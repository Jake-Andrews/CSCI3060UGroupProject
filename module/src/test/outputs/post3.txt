Welcome, please login to use the program. 
To exit type exit, To start type login: 
Please enter a valid username: 
You have been logged in!

Please enter a command!
For a list of commands type 'help': Please enter the city where the unit you are renting is: 
Please enter a valid price to rent the unit at (max is 999.99): 
Please enter the number of bedrooms (max is 9): 
Please enter the number of bedrooms (max is 9): 
Creating unit now!
Daily transaction file string: 03_USER___________AA_b553d60d_Toronto___________________5_500.0_00

Please enter a command!
For a list of commands type 'help': Daily transaction file string: 00_USER___________AA____________________________0_0.0___00

Printing Daily Transactions:03_USER___________AA_b553d60d_Toronto___________________5_500.0_00
00_USER___________AA____________________________0_0.0___00
You have been successfully logged out!
Welcome, please login to use the program. 
To exit type exit, To start type login: Thank you for using our program!