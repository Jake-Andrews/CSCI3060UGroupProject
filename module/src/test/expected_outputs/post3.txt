Welcome, please login to use the program. 
To exit type exit, To start type login:
Please enter a valid username:
You have been logged in!

Please enter a command!
For a list of commands type 'help':
Please enter the city where the unit you are renting is:

Please enter a valid price to rent the unit at (max is 999.99):

Please enter the number of bedrooms (max is 9):

Please enter the number of bedrooms (max is 9): 

Creating unit now!
Daily transaction file string: 03_USER_____________AA__(ID)______Toronto____________________5__500.0___0

Please enter a command!
For a list of commands type 'help':
Daily transaction file string: 00_USER_____________AA_______________________________0__0.0_____0

Printing Daily Transactions:03_USER_____________AA__(ID)______Toronto____________________5__500.0___0
00_USER_____________AA_______________________________0__0.0_____0
You have been successfully logged out!
Welcome, please login to use the program.
To exit type exit, To start type login:
Thank you for using our program!